### PR TITLE
purplehuman's first Goodie

### PR DESCRIPTION
**Pull request by purplehuman**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
I did the first tutorial, and when you search "duckduckhack purplehuman", it says "purplehuman is awesome!"

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It says "purplehuman is awesome!" ! No other search engine would say that, would they?

**What is the data source for your Instant Answer? (Provide a link if possible)**
Hmm, me.

**Why did you choose this data source?**
Err, because Instant Answer is about me.

**Are there any other alternative (better) data sources?**
I don't think so, I'm not sure though.

**What are some example queries that trigger this Instant Answer?**
"duckduckhack purplehuman", "duckduckhack purplehuman is WHAT?"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
OMG, just me.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Not really.

**Which existing Instant Answers will this one supercede/overlap with?**
None.

**Are you having any problems? Do you need our help with anything?**
Thanks. Not yet anyway.

**Where did you hear about DuckDuckHack? (For first time contributors)**
I think I was searching for open source projects to contribute on DuckDuckGo.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot from 2015-01-04 06 21 55

## Checklist
Please place an 'X' where appropriate.

```
[] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/purplehuman.pm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/purplehuman.pm";
+attribution github => ["https://github.com/purplehuman", "purplehuman"];
+
+triggers start => "duckduckhack purplehuman";
+
+handle remainder => sub {
+    return if $_;
+	return "purplehuman is awesome!";
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/purplehuman.pm]: This should be `purplehuman is awesome and has successfully completed the DuckDuckHack Goodie tutorial!`, *please also indent with 4 spaces like the above return line* don't forget to update your test file!

**Comment by MrChrisW** [comment]
@purplehuman Congratulations on your first Goodie  (cake), I've added a little bit of feedback, once  updated this should be good to go.

**Comment by purplehuman** [comment]
Thanks for the quick reply. I made the necessary changes.

**Comment by purplehuman** [comment]
Sorry I forgot the test file.

**Comment by mintsoft** [comment]
@purplehuman Thanks for the fast turnaround on the fixes; congratulations on your first goodie! :cake:

**Comment by mintsoft** [comment]
@MrChrisW Thanks for the review!

**Comment by moollaza** [comment]
@ghost it's live! https://duckduckgo.com/?q=duckduckhack+purplehuman&t=ffab&ia=answer

I see you've changed your GitHub username now...care to update the Goodie?

**Comment by moollaza** [comment]
Now that you've completed your first Goodie, we really hope you'll work on another! Feel free to check out our [Ideas Forum](https://duck.co/ideas) or look through our Github issues if you need something to work on!

**Pull request merged**